### ISSUE TITLE
storage_proxy - compressed chunk failed checksum

### ISSUE THREAD

**Issue by larisau** [scylladb/scylla#3019]
*Installation details*
Scylla version (or git commit hash):
https://s3.amazonaws.com/downloads.scylladb.com/rpm/unstable/centos/branch-2.0/29/scylla.repo
Cluster size: 1 node 
Hardware details:
gce instance n1-standard-32: 32 cpu-s, 120 GB mem, 3 TB SSD( 8 disks, was not intended but happened)

preparation write:
cassandra-stress write n=100000000 -port jmx=6868 -mode cql3 native -rate threads=100 -col 'size=FIXED(1024) n=FIXED(1)' -pop seq=1..100000000 -log interval=5

Run cassandra stress read by 8 loaders 2 threads each.
read cmd:
cassandra-stress read duration=10080m -port jmx=6868 -mode cql3 native -rate threads=500 -col 'size=FIXED(1024) n=FIXED(1)' -pop seq=1..100000000 -log interval=5

About 3 hours after start started to get read timeouts.

` DEBUG| [35.190.128.64] [stdout] total,     153973936,    4116,    4116,    4116,   122.7,     0.9,    12.2,  3894.6,  3938.6,  3953.2,12594.4,  0.00700,      0,      0,       0,       0,       0,       0
2017-11-26 21:10:26,722 remote           L0197 DEBUG| [35.196.91.251] [stdout] total,     153977537,    4217,    4217,    4217,   119.1,     0.7,    12.5,  3892.2,  3938.0,  3952.6,12473.9,  0.00509,      0,      0,       0,       0,       0,       0
2017-11-26 21:10:26,811 remote           L0197 DEBUG| [104.196.190.169] [stdout] total,     153968602,    4507,    4507,    4507,   122.6,     0.9,    11.6,  3902.6,  3932.4,  3955.2,12260.1,  0.00492,      0,      0,       0,       0,       0,       0
2017-11-26 21:10:26,986 remote           L0197 DEBUG| [35.190.159.217] [stdout] Nov 26 21:10:26 err    |  [shard 14] storage_proxy - Exception when communicating with 10.240.0.16: sstable inactive read queue overloaded
2017-11-26 21:10:26,986 remote           L0197 DEBUG| [35.190.159.217] [stdout] Nov 26 21:10:26 err    |  [shard 21] storage_proxy - Exception when communicating with 10.240.0.16: sstable inactive read queue overloaded
2017-11-26 21:10:26,987 remote           L0197 DEBUG| [35.190.159.217] [stdout] Nov 26 21:10:26 err    |  [shard 21] storage_proxy - Exception when communicating with 10.240.0.16: sstable inactive read queue overloaded
2017-11-26 21:10:26,990 remote           L0197 DEBUG| [35.190.159.217] [stdout] Nov 26 21:10:26 err    |  [shard 8] storage_proxy - Exception when communicating with 10.240.0.16: sstable inactive read queue overloaded
2017-11-26 21:10:26,991 remote           L0197 DEBUG| [35.190.159.217] [stdout] Nov 26 21:10:26 err    |  [shard 6] storage_proxy - Exception when communicating with 10.240.0.16: sstable inactive read queue overloaded
2017-11-26 21:10:26,992 remote           L0197 DEBUG| [35.190.159.217] [stdout] Nov 26 21:10:26 err    |  [shard 31] storage_proxy - Exception when communicating with 10.240.0.16: sstable inactive read queue overloaded
2017-11-26 21:10:26,993 remote           L0197 DEBUG| [35.190.159.217] [stdout] Nov 26 21:10:26 err    |  [shard 20] storage_proxy - Exception when communicating with 10.240.0.16: sstable inactive read queue overloaded
2017-11-26 21:10:26,993 remote           L0197 DEBUG| [35.190.159.217] [stdout] Nov 26 21:10:26 err    |  [shard 30] storage_proxy - Exception when communicating with 10.240.0.16: sstable inactive read queue overloaded
2017-11-26 21:10:26,994 remote           L0197 DEBUG| [35.190.159.217] [stdout] Nov 26 21:10:26 err    |  [shard 23] storage_proxy - Exception when communicating with 10.240.0.16: sstable inactive read queue overloaded
2017-11-26 21:10:26,995 remote           L0197 DEBUG| [35.190.159.217] [stdout] Nov 26 21:10:26 err    |  [shard 10] storage_proxy - Exception when communicating with 10.240.0.16: sstable inactive read queue overloaded
2017-11-26 21:10:26,998 remote           L0197 DEBUG| [35.190.159.217] [stdout] Nov 26 21:10:26 err    |  [shard 6] storage_proxy - Exception when communicating with 10.240.0.16: sstable inactive read queue overloaded
2017-11-26 21:10:27,000 remote           L0197 DEBUG| [35.190.159.217] [stdout] Nov 26 21:10:26 err    |  [shard 16] storage_proxy - Exception when communicating with 10.240.0.16: sstable inactive read queue overloaded
2017-11-26 21:10:27,066 remote           L0197 DEBUG| [35.190.159.217] [stdout] Nov 26 21:10:26 err    |  [shard 14] storage_proxy - Exception when communicating with 10.240.0.16: sstable inactive read queue overloaded
2017-11-26 21:10:27,066 remote           L0197 DEBUG| [35.190.159.217] [stdout] Nov 26 21:10:26 err    |  [shard 7] storage_proxy - Exception when communicating with 10.240.0.16: sstable inactive read queue overloaded
2017-11-26 21:10:27,066 remote           L0197 DEBUG| [35.190.159.217] [stdout] Nov 26 21:10:26 err    |  [shard 2] storage_proxy - Exception when communicating with 10.240.0.16: sstable inactive read queue overloaded
2017-11-26 21:10:27,066 remote           L0197 DEBUG| [35.190.159.217] [stdout] Nov 26 21:10:26 err    |  [shard 25] storage_proxy - Exception when communicating with 10.240.0.16: sstable inactive read queue overloaded
2017-11-26 21:10:28,434 remote           L0197 DEBUG| [104.196.204.146] [stdout] total,     153986144,    4732,    4732,    4732,   120.0,     0.8,     3.8,  3935.9,  3972.5,  3999.0,12229.6,  0.00359,      0,      0,`
............................
............................
`2017-11-27 09:14:27,649 remote           L0197 DEBUG| [35.190.159.217] [stdout] Nov 27 09:14:27 err    |  [shard 16] storage_proxy - Exception when communicating with 10.240.0.16: compressed chunk failed checksum
2017-11-27 09:17:10,612 remote           L0197 DEBUG| [35.190.159.217] [stdout] Nov 27 09:17:10 err    |  [shard 16] storage_proxy - Exception when communicating with 10.240.0.16: compressed chunk failed checksum
2017-11-27 09:21:20,708 remote           L0197 DEBUG| [35.190.159.217] [stdout] Nov 27 09:21:20 err    |  [shard 16] storage_proxy - Exception when communicating with 10.240.0.16: compressed chunk failed checksum
`

**Comment by larisau** [comment]
journalctl -t scylla -e
`Nov 27 10:52:16 load-test-db-node-a29ba94a-0-1 scylla[1403]:  [shard 11] sstable - reader 0x60b0d6962300: reading partition
Nov 27 10:52:16 load-test-db-node-a29ba94a-0-1 scylla[1403]:  [shard 11] sstable - index 0x60b000c7fe80: index_reader for /var/lib/scylla/data/system/local-7ad54392bcdd35a684174e047860b377/system-local-ka-171-Data.db
Nov 27 10:52:16 load-test-db-node-a29ba94a-0-1 scylla[1403]:  [shard 11] sstable - index 0x60b0d74b8640: index_reader for /var/lib/scylla/data/system/local-7ad54392bcdd35a684174e047860b377/system-local-ka-171-Data.db
Nov 27 10:52:16 load-test-db-node-a29ba94a-0-1 scylla[1403]:  [shard 11] sstable - index 0x60b0d74b8640: advance_to({0x60b000cf52f8, w=-1}), _previous_summary_idx=0, _current_summary_idx=0
Nov 27 10:52:16 load-test-db-node-a29ba94a-0-1 scylla[1403]:  [shard 11] sstable - index 0x60b0d74b8640: first entry
Nov 27 10:52:16 load-test-db-node-a29ba94a-0-1 scylla[1403]:  [shard 11] sstable - index 0x60b000c7fe80: advance_to({0x60b000cf52c8, w=-1}), _previous_summary_idx=0, _current_summary_idx=0
Nov 27 10:52:16 load-test-db-node-a29ba94a-0-1 scylla[1403]:  [shard 11] sstable - index 0x60b000c7fe80: first entry
Nov 27 10:52:16 load-test-db-node-a29ba94a-0-1 scylla[1403]:  [shard 11] sstable - reader 0x60b0d6962300: reading partition
Nov 27 10:52:16 load-test-db-node-a29ba94a-0-1 scylla[1403]:  [shard 11] sstable - index 0x60b0d74b8640: index_reader for /var/lib/scylla/data/system/local-7ad54392bcdd35a684174e047860b377/system-local-ka-171-Data.db
Nov 27 10:52:16 load-test-db-node-a29ba94a-0-1 scylla[1403]:  [shard 11] sstable - index 0x60b000c7fe80: index_reader for /var/lib/scylla/data/system/local-7ad54392bcdd35a684174e047860b377/system-local-ka-171-Data.db
Nov 27 10:52:16 load-test-db-node-a29ba94a-0-1 scylla[1403]:  [shard 11] sstable - index 0x60b000c7fe80: advance_to({0x60b000cf52f8, w=-1}), _previous_summary_idx=0, _current_summary_idx=0
Nov 27 10:52:16 load-test-db-node-a29ba94a-0-1 scylla[1403]:  [shard 11] sstable - index 0x60b000c7fe80: first entry
Nov 27 10:52:16 load-test-db-node-a29ba94a-0-1 scylla[1403]:  [shard 11] sstable - index 0x60b0d74b8640: advance_to({0x60b000cf52c8, w=-1}), _previous_summary_idx=0, _current_summary_idx=0
Nov 27 10:52:16 load-test-db-node-a29ba94a-0-1 scylla[1403]:  [shard 11] sstable - index 0x60b0d74b8640: first entry
Nov 27 10:52:16 load-test-db-node-a29ba94a-0-1 scylla[1403]:  [shard 11] sstable - reader 0x60b0d6962300: reading partition
Nov 27 10:52:16 load-test-db-node-a29ba94a-0-1 scylla[1403]:  [shard 11] sstable - index 0x60b000c7fe80: index_reader for /var/lib/scylla/data/system/local-7ad54392bcdd35a684174e047860b377/system-local-ka-171-Data.db
Nov 27 10:52:16 load-test-db-node-a29ba94a-0-1 scylla[1403]:  [shard 11] sstable - index 0x60b0d74b8640: index_reader for /var/lib/scylla/data/system/local-7ad54392bcdd35a684174e047860b377/system-local-ka-171-Data.db
Nov 27 10:52:16 load-test-db-node-a29ba94a-0-1 scylla[1403]:  [shard 11] sstable - index 0x60b0d74b8640: advance_to({0x60b000cf52f8, w=-1}), _previous_summary_idx=0, _current_summary_idx=0
Nov 27 10:52:16 load-test-db-node-a29ba94a-0-1 scylla[1403]:  [shard 11] sstable - index 0x60b0d74b8640: first entry
Nov 27 10:52:16 load-test-db-node-a29ba94a-0-1 scylla[1403]:  [shard 11] sstable - index 0x60b000c7fe80: advance_to({0x60b000cf52c8, w=-1}), _previous_summary_idx=0, _current_summary_idx=0
Nov 27 10:52:16 load-test-db-node-a29ba94a-0-1 scylla[1403]:  [shard 11] sstable - index 0x60b000c7fe80: first entry
Nov 27 10:52:16 load-test-db-node-a29ba94a-0-1 scylla[1403]:  [shard 11] sstable - reader 0x60b0d6962300: reading partition
Nov 27 10:52:16 load-test-db-node-a29ba94a-0-1 scylla[1403]:  [shard 11] sstable - index 0x60b0d74b8640: index_reader for /var/lib/scylla/data/system/local-7ad54392bcdd35a684174e047860b377/system-local-ka-171-Data.db
Nov 27 10:52:16 load-test-db-node-a29ba94a-0-1 scylla[1403]:  [shard 1] storage_proxy - Exception when communicating with 10.240.0.16: compressed chunk failed checksum
Nov 27 10:52:16 load-test-db-node-a29ba94a-0-1 scylla[1403]:  [shard 2] storage_proxy - Exception when communicating with 10.240.0.16: compressed chunk failed checksum
Nov 27 10:52:17 load-test-db-node-a29ba94a-0-1 scylla[1403]:  [shard 3] storage_proxy - Exception when communicating with 10.240.0.16: compressed chunk failed checksum
Nov 27 10:52:17 load-test-db-node-a29ba94a-0-1 scylla[1403]:  [shard 4] storage_proxy - Exception when communicating with 10.240.0.16: compressed chunk failed checksum
Nov 27 10:52:26 load-test-db-node-a29ba94a-0-1 scylla[1403]:  [shard 0] storage_proxy - Exception when communicating with 10.240.0.16: compressed chunk failed checksum
Nov 27 10:52:26 load-test-db-node-a29ba94a-0-1 scylla[1403]:  [shard 1] storage_proxy - Exception when communicating with 10.240.0.16: compressed chunk failed checksum
Nov 27 10:52:26 load-test-db-node-a29ba94a-0-1 scylla[1403]:  [shard 2] storage_proxy - Exception when communicating with 10.240.0.16: compressed chunk failed checksum
Nov 27 10:52:26 load-test-db-node-a29ba94a-0-1 scylla[1403]:  [shard 3] storage_proxy - Exception when communicating with 10.240.0.16: compressed chunk failed checksum
Nov 27 10:52:27 load-test-db-node-a29ba94a-0-1 scylla[1403]:  [shard 4] storage_proxy - Exception when communicating with 10.240.0.16: compressed chunk failed checksum
Nov 27 10:52:27 load-test-db-node-a29ba94a-0-1 scylla[1403]:  [shard 5] storage_proxy - Exception when communicating with 10.240.0.16: compressed chunk failed checksum
Nov 27 10:52:27 load-test-db-node-a29ba94a-0-1 scylla[1403]:  [shard 6] storage_proxy - Exception when communicating with 10.240.0.16: compressed chunk failed checksum
Nov 27 10:52:28 load-test-db-node-a29ba94a-0-1 scylla[1403]:  [shard 7] storage_proxy - Exception when communicating with 10.240.0.16: compressed chunk failed checksum
Nov 27 10:52:28 load-test-db-node-a29ba94a-0-1 scylla[1403]:  [shard 8] storage_proxy - Exception when communicating with 10.240.0.16: compressed chunk failed checksum
Nov 27 10:52:30 load-test-db-node-a29ba94a-0-1 scylla[1403]:  [shard 9] storage_proxy - Exception when communicating with 10.240.0.16: compressed chunk failed checksum
Nov 27 10:52:30 load-test-db-node-a29ba94a-0-1 scylla[1403]:  [shard 10] storage_proxy - Exception when communicating with 10.240.0.16: compressed chunk failed checksum
`

**Comment by larisau** [comment]
cqlsh cannot connect:

cqlsh 10.240.0.16
Connection error: ('Unable to connect to any servers', {'10.240.0.16': OperationTimedOut('errors=None, last_host=None',)})

**Comment by glommer** [comment]
Finally we saw this on our QA!

Guys, do we already have Raphael's patches ?

**Comment by larisau** [comment]
[job_log.tar.gz](https://github.com/scylladb/scylla/files/1505567/job_log.tar.gz)

**Comment by avikivity** [comment]
```
hexdump -C /var/lib/scylla/data/system/local-7ad54392bcdd35a684174e047860b377/system-local-ka-171-CompressionInfo.db
00000000  00 0d 4c 5a 34 43 6f 6d  70 72 65 73 73 6f 72 00  |..LZ4Compressor.|
00000010  00 00 01 00 10 63 72 63  5f 63 68 65 63 6b 5f 63  |.....crc_check_c|
00000020  68 61 6e 63 65 00 03 31  2e 30 00 00 10 00 00 00  |hance..1.0......|
00000030  00 00 00 00 31 9b 00 00  00 04 00 00 00 00 00 00  |....1...........|
00000040  00 00 00 00 00 00 00 00  08 7c 00 00 00 00 00 00  |.........|......|
00000050  10 01 00 00 00 00 00 00  17 96                    |..........|
0000005a
```

**Comment by avikivity** [comment]
Decoded, 4 offsets: 0, 0x87c, 0x1001, 0x1796 (decimal: 0, 2172, 4097, 6038)

**Comment by avikivity** [comment]
data_length is 0x319b (12699). So 4 chunks.

**Comment by avikivity** [comment]
System tables from that node:

[3019-system-tables.tar.gz](https://github.com/scylladb/scylla/files/1505647/3019-system-tables.tar.gz)

**Comment by slivne** [comment]
Its without Raphael's patches ....

scylla-conf-2.0.0-0.20170929.e265c91.el7.centos.x86_64
scylla-server-2.0.0-0.20170929.e265c91.el7.centos.x86_64
scylla-tools-2.0.0-20170930.81c4784.el7.centos.noarch
scylla-tools-core-2.0.0-20170930.81c4784.el7.centos.noarch
scylla-kernel-conf-2.0.0-0.20170929.e265c91.el7.centos.x86_64
scylla-2.0.0-0.20170929.e265c91.el7.centos.x86_64
scylla-gdb-7.10.1-30.el7.centos.x86_64
scylla-debuginfo-2.0.0-0.20170929.e265c91.el7.centos.x86_64
scylla-jmx-2.0.0-20170930.e9c546a.el7.centos.noarch
scylla-env-1.0-1.el7.centos.noarch

**Comment by avikivity** [comment]
Some sstables are missing their Summary.db, even though their TOC.txt mentions it.

**Comment by raphaelsc** [comment]
```
-rw-r--r--. 1 utroz utroz   82 Nov 26 14:38 system_schema-columns-ka-180-CompressionInfo.db
-rw-r--r--. 1 utroz utroz 2.4K Nov 26 14:38 system_schema-columns-ka-180-Data.db
-rw-r--r--. 1 utroz utroz   10 Nov 26 14:38 system_schema-columns-ka-180-Digest.sha1
-rw-r--r--. 1 utroz utroz 1.3K Nov 26 14:38 system_schema-columns-ka-180-Filter.db
-rw-r--r--. 1 utroz utroz   27 Nov 26 14:38 system_schema-columns-ka-180-Index.db
-rw-r--r--. 1 utroz utroz   38 Nov 26 14:38 system_schema-columns-ka-180-Scylla.db
-rw-r--r--. 1 utroz utroz 4.5K Nov 26 14:38 system_schema-columns-ka-180-Statistics.db
-rw-r--r--. 1 utroz utroz   99 Nov 27 09:15 system_schema-columns-ka-180-Summary.db
-rw-r--r--. 1 utroz utroz  101 Nov 26 14:38 system_schema-columns-ka-180-TOC.txt
```

looks like all missing summary is from this column family, and note that this summary was built only one day later, we have the ability to do that. One thing that comes to my mind is our change to dynamically generate summary entries. But I don't see how that would cause it, we should generate at least one.

**Comment by raphaelsc** [comment]
oops. that change hasn't even reached e265c91 (2.0.0) yet.

**Comment by slivne** [comment]
Can we correlate the time of the creation of the summary file with
something that happend in the cluster - was the node restarted etc.

what happend around this time ?

Nov 27 09:15 system_schema-columns-ka-180-Summary.db

**Comment by gleb-cloudius** [comment]
On Mon, Nov 27, 2017 at 02:32:48PM +0000, Shlomi Livne wrote:
> Can we correlate the time of the creation of the summary file with
> something that happend in the cluster - was the node restarted etc.
> 
AFAIK this is one node cluster.

--
			Gleb.

**Comment by slivne** [comment]
On Mon, Nov 27, 2017 at 4:34 PM, Gleb Natapov <notifications@github.com>
wrote:

> On Mon, Nov 27, 2017 at 02:32:48PM +0000, Shlomi Livne wrote:
> > Can we correlate the time of the creation of the summary file with
> > something that happend in the cluster - was the node restarted etc.
>

something that happen on the node -  did we restart the node ? did it crash
and restart ?

**Comment by avikivity** [comment]
Comparing two Index.db:

```
$ hexdump -C  /var/lib/scylla/data/system_schema/columns-24101c25a2ae3af787c1b40ee1aca33f/system_schema-columns-ka-212-Index.db
00000000  00 00 00 00 00 00 00 00  00 00 00 00 00 00 00 00  |................|
00000010  00 00 00 00 00 00 00 00  00 00 00                 |...........|
0000001b
$ hexdump -C  /var/lib/scylla/data/system_schema/columns-24101c25a2ae3af787c1b40ee1aca33f/system_schema-columns-ka-180-Index.db 
00000000  00 0d 73 79 73 74 65 6d  5f 74 72 61 63 65 73 00  |..system_traces.|
00000010  00 00 00 00 00 00 00 00  00 00 00                 |...........|
0000001b
```

**Comment by avikivity** [comment]
One is clearly bad.

**Comment by raphaelsc** [comment]
Good to know. Note that we wrote the bytes and then truncated. We're either losing a buffer in Scylla itself or it could be that race in XFS which zeroes EOF block but that should only happen if extending the file which is never our case because we truncate the file first. Could be a related unknown unknown in XFS too.

**Comment by raphaelsc** [comment]
I think it would be good to ask @larisau to enable integrity check interposer and try to reproduce it again.

**Comment by raphaelsc** [comment]
checked that there's no sporadic 4k unwritten extent in the file:
```
# xfs_bmap -vvp system_schema-columns-ka-212-Index.db 
system_schema-columns-ka-212-Index.db:
 EXT: FILE-OFFSET      BLOCK-RANGE          AG AG-OFFSET        TOTAL FLAGS
   0: [0..7]:          589826064..589826071  3 (204816..204823)     8 01111
 FLAG Values:
    010000 Unwritten preallocated extent
    001000 Doesn't begin on stripe unit
    000100 Doesn't end   on stripe unit
    000010 Doesn't begin on stripe width
    000001 Doesn't end   on stripe width
```
@avikivity 
we're not extending file on writes, so i don't see how XFS would be triggering EOF block zeroing. As I said, either an unknown-unknown issue in XFS, or a Scylla bug. The latter is more likely. However, it's a really rare issue which makes me think it's either a disk and/or XFS issue involving race condition.

**Comment by raphaelsc** [comment]
@avikivity Some data file is completely zeroed as well. I think the course of action is to provide Scylla 2.0 + integrity checker feature for testing and try to reproduce the issue again. From sstables alone, we can't figure out much.

**Comment by avikivity** [comment]
@raphaelsc which file data?

**Comment by raphaelsc** [comment]
On Tue, Nov 28, 2017 at 7:36 AM, Avi Kivity <notifications@github.com>
wrote:

> @raphaelsc <https://github.com/raphaelsc> which file data?
>
system_schema-columns-ka-93-Data.db
and system_schema-columns-ka-212-Data.db. they both contain missing summary
and zeroed index too.

**Comment by avikivity** [comment]
That's super strange. Maybe we have a bug writing really small files, but that would show up often because it always happens in system tables.

**Comment by raphaelsc** [comment]
On Tue, Nov 28, 2017 at 8:41 AM, Avi Kivity <notifications@github.com>
wrote:

> That's super strange. Maybe we have a bug writing really small files, but
> that would show up often because it always happens in system tables.
>
Indeed. I think the best course of action is to enable integrity checker
and try to reproduce it again. It just took a few hours for Larisa to do
it. The strangest thing for me so far is the missing summary. I have no
idea how that could have happened given that no summary entry is even
needed (just header).

**Comment by slivne** [comment]
AFAIK we do not have the integrity in 2.0 branch

We can try and redo this with 2.1 branch that has it AFAIK.

I'll ask Larisa to return with that and enable the integrity checker.

Please note that compaction has stopped long ago and reads have been
running for a long while ...

In my view - we still do not have an answer to a piece of the puzzle if the
timestamps of files are different (in days) it would be good to explain
what may have happend that could trigger that.

On Tue, Nov 28, 2017 at 12:56 PM, Raphael Carvalho <notifications@github.com

**Comment by raphaelsc** [comment]
On Tue, Nov 28, 2017 at 9:04 AM, Shlomi Livne <notifications@github.com>
wrote:

> AFAIK we do not have the integrity in 2.0 branch


> We can try and redo this with 2.1 branch that has it AFAIK.
>
> I'll ask Larisa to return with that and enable the integrity checker.
>

Sure. thanks.


>
> Please note that compaction has stopped long ago and reads have been
> running for a long while ...
>
> In my view - we still do not have an answer to a piece of the puzzle if the
> timestamps of files are different (in days) it would be good to explain
> what may have happend that could trigger that.


That only can be explained at time of write, thus integrity checker. The
million-dollar question is: Why did we generate a table without a summary?
The timestamp for summary is different because we can regenerate summary on
the fly if scylla finds it to be missing.

**Comment by slivne** [comment]
we are not able to reproduce this - since the issue has been reported we have fixed scylladb/seastar@fe19c876c34fc99e4e79a02e7f45ec43acb6921b
